### ISSUE TITLE
Does TensorRT support Batch Normalization?

### ISSUE THREAD

**Issue by tengfeixing** [dusty-nv/jetson-inference#28]
Hi, I have re-trained DetectNet with batch normalization layers , but I failed to run TensorRT over this caffe model.
When load the model file, here shows some error infos:
`Message type "diccaffe.BatchNormParameter" has no filed name "scale_filler", could not parse deploy file.`
After I delete the batch_norm_param filed in the deploy file, it shows another error info:
`caffeParser.cpp:613: bool bnConvert(const nvinfer1::Weights&, const nvinfer1::Weights&, const nvinfer1::Weights&, float, nvinfer1::Weights&, nvinfer1::Weights&, std::vector<void*>&) [with T = float]: Assertion `mean.count == variance.count && movingAverage.count == 1' failed.`

It will be appreciated if you can give me some advice.

**Comment by dusty-nv** [comment]
Hi there, I have confirmed that TensorRT does indeed support batch normalization.   I think it may only be the scale_filler parameter that isn't.  Have you tried removing only that from your prototxt?
